controller/search_blackbox_test.go

```diff
@@ -519,81 +397,54 @@ func (s *searchBlackBoxTest) TestSearchFilter() {
-		fxt.WorkItems[idx].Type = workitem.SystemBug
-		return nil
-	}))
-
-	tf.NewTestFixture(s.T(), s.DB, tf.WorkItems(5, func(fxt *tf.TestFixture, idx int) error {
-		fxt.WorkItems[idx].Fields[workitem.SystemTitle] = fmt.Sprintf("Closed issue #%d", idx)
-		fxt.WorkItems[idx].Fields[workitem.SystemState] = workitem.SystemStateClosed
-		fxt.WorkItems[idx].Fields[workitem.SystemIteration] = sprint2.ID.String()
-		fxt.WorkItems[idx].Fields[workitem.SystemLabels] = []string{lblUI.ID.String()}
-		fxt.WorkItems[idx].Fields[workitem.SystemAssignees] = []string{bob.ID.String()}
-		fxt.WorkItems[idx].Fields[workitem.SystemCreator] = s.testIdentity.ID.String()
-		fxt.WorkItems[idx].SpaceID = sprint2.SpaceID
-		fxt.WorkItems[idx].Type = workitem.SystemFeature
-		return nil
-	}))
+	// given
+	fxt := tf.NewTestFixture(s.T(), s.DB,
+		tf.Identities(3, tf.SetIdentityUsernames([]string{"spaceowner", "alice", "bob"})),
+		tf.Iterations(2, tf.SetIterationNames([]string{"sprint1", "sprint2"})),
+		tf.Labels(4, tf.SetLabelNames([]string{"important", "backend", "ui", "rest"})),
+		tf.WorkItemTypes(2, tf.SetWorkItemTypeNames([]string{"bug", "feature"})),
+		tf.WorkItems(3+5+1, func(fxt *tf.TestFixture, idx int) error {
+			wi := fxt.WorkItems[idx]
+			if idx < 3 {
+				//wi.Fields[workitem.SystemTitle] = fmt.Sprintf("New issue #%d", idx)
+				wi.Fields[workitem.SystemState] = workitem.SystemStateResolved
+				wi.Fields[workitem.SystemIteration] = fxt.IterationByName("sprint1").ID.String()
+				wi.Fields[workitem.SystemLabels] = []string{fxt.LabelByName("important").ID.String(), fxt.LabelByName("backend").ID.String()}
+				wi.Fields[workitem.SystemAssignees] = []string{fxt.IdentityByUsername("alice").ID.String()}
+				wi.Fields[workitem.SystemCreator] = fxt.IdentityByUsername("spaceowner").ID.String()
+				wi.Type = fxt.WorkItemTypeByName("bug").ID
+			} else if idx < 3+5 {
+				//wi.Fields[workitem.SystemTitle] = fmt.Sprintf("Closed issue #%d", idx)
+				wi.Fields[workitem.SystemState] = workitem.SystemStateClosed
+				wi.Fields[workitem.SystemIteration] = fxt.IterationByName("sprint2").ID.String()
+				wi.Fields[workitem.SystemLabels] = []string{fxt.LabelByName("ui").ID.String()}
+				wi.Fields[workitem.SystemAssignees] = []string{fxt.IdentityByUsername("bob").ID.String()}
+				wi.Fields[workitem.SystemCreator] = fxt.IdentityByUsername("spaceowner").ID.String()
+				wi.Type = fxt.WorkItemTypeByName("feature").ID
+			} else {
+				// wi.Fields[workitem.SystemTitle] = fmt.Sprintf("Unassigned issue")
+				wi.Fields[workitem.SystemState] = workitem.SystemStateClosed
+				wi.Fields[workitem.SystemIteration] = fxt.IterationByName("sprint2").ID.String()
+				wi.Fields[workitem.SystemCreator] = fxt.IdentityByUsername("spaceowner").ID.String()
+				wi.Type = fxt.WorkItemTypeByName("feature").ID
+			}
+			return nil
+		}),
+	)
+	spaceIDStr := fxt.WorkItems[0].SpaceID.String()
 
 	s.T().Run("label IN IMPORTAND, UI", func(t *testing.T) {
```

Comment:
I know it's not yours 
s/IMPORTAND/IMPORTANT

Reply:
Fixed in be474a7. Thanks